### ISSUE TITLE
Option smartbanner:hide-path not working

### ISSUE THREAD

**Issue by tiasmc** [ain/smartbanner.js#80]
__Steps to reproduce__: (mostly the same as #71 )

1. add meta tag <meta name="smartbanner:hide-path" content="/">
2. go to a page with the banner that is not the root page of the site. Eg: example.com/test/path/page
3. hide the banner and refresh
4. the banner is not shown and the smartbanner_exited cookie is set correctly accept for the path. The path is set to /test/path/page
5. Go to a page that is lower in the page tree. Eg: example.com/test or example.com and the banner will popup again

__What happens__:
In smartbanner.js line: 201 "Bakery.bake(this.hideTtl, this.hidePath);"
this.hidePath is undefined. Then the default path (current page) is used.

__What should happen__:
this.hidePath should have the "/" value from the meta tag?

It seems this.options.hidePath is filled with the correct content from the meta tag.

**Comment by ain** [comment]
Thanks for the report. Please post the exact block of meta tags your are using.

**Comment by tiasmc** [comment]
Yes ofcourse, here they are:
```
<meta name="smartbanner:title" content="My App" />
<meta name="smartbanner:author" content="My Author" />
<meta name="smartbanner:price" content="GRATIS" />
<meta name="smartbanner:price-suffix-google" content=" - Hent den i Google Play" />
<meta name="smartbanner:icon-google" content="smartbanner-icon.png"/>
<meta name="smartbanner:button" content="Vis" />
<meta name="smartbanner:button-url-google" content="https://play.google.com/store/..."/>
<meta name="smartbanner:enable-platform" content="android"/>
<meta name="smartbanner:exclude-user-agent-regex" content="iPhone|iPad"/>
<meta name="smartbanner:disable-positioning" content="true">
<meta name="smartbanner:hide-path" content="/">
<meta name="smartbanner:custom-design-modifier" content="ios">
```

**Comment by ivensalves** [comment]
The issue here is that there is no code to retrieve the meta name="smartbanner:hide-path" on the JS code. I added the following code

```
                }, {
                key: 'hidePath',
                get: function get() {
                    return this.options.hidePath;
                }
```

inside the `_createClass(SmartBanner, [{` call.

**Comment by PavelK27** [comment]
Can confirm having the same problem with that option. And can also confirm fix from @ivensalves Thanks!

**Comment by ain** [comment]
Fixed in [v1.10.1](https://github.com/ain/smartbanner.js/releases/tag/v1.10.1)
